### ISSUE TITLE
Missing type declaration file for plugin alias

### ISSUE THREAD

**Issue by duduluu** [rollup/plugins#295]
- Rollup Plugin Name: alias
- Rollup Plugin Version: 3.0.1
- Rollup Version:
- Operating System (or Browser):
- Node Version: 

### How Do We Reproduce?

Checkout package.json on npm
https://cdn.jsdelivr.net/npm/@rollup/plugin-alias@3.0.1/package.json

```json
{
  "name": "@rollup/plugin-alias",
  "version": "3.0.1",

  "files": [
    "dist",
    "src",
    "README.md",
    "LICENSE"
  ],

}
```

Event it's correct in the source code.

### Expected Behavior

Includes directory `types` in package.json field `files`.

### Actual Behavior

Incorrectly includes directory `src` but `types` in package.json field `files`.

**Comment by shellscape** [comment]
Thanks when we get the chance to republish we'll clear this up.

**Comment by shellscape** [comment]
Fixed in #228
